examples/postgresql-multiple-datasources/helm.yaml

```diff
@@ -0,0 +1,53 @@
+build:
+  uri: https://github.com/wildfly/wildfly-s2i
+  mode: s2i
+  contextDir: examples/postgresql-multiple-datasources
+  ref: main
+  s2i:
+    version: latest
+    builderImage: quay.io/wildfly/wildfly-s2i-jdk11
+    runtimeImage: quay.io/wildfly/wildfly-runtime-jdk11
+deploy:
+  replicas: 1
```

Comment:
Value of 1 is the default, you can remove this field.

Reply:
Done, removed these items as they are the default values.